builtInFunctions/esdtMetaDataUpdate.go

```diff
@@ -142,6 +149,14 @@ func (e *esdtMetaDataUpdate) ProcessBuiltinFunction(acntSnd, _ vmcommon.UserAcco
 		ReturnCode:   vmcommon.Ok,
 		GasRemaining: vmInput.GasProvided - gasToUse,
 	}
+
+	esdtDataBytes, err := e.marshaller.Marshal(esdtInfo.esdtData)
+	if err != nil {
+		log.Warn("esdtMetaDataUpdate.ProcessBuiltinFunction: cannot marshall esdt data for log", "error", err)
+	}
+
+	addESDTEntryInVMOutput(vmOutput, []byte(core.ESDTMetaDataUpdate), vmInput.Arguments[0], esdtInfo.esdtData.TokenMetaData.Nonce, big.NewInt(0), vmInput.CallerAddr, esdtDataBytes)
```

Comment:
is it safe to pass `esdtDataBytes` as nil here in case of marshall error?

Reply:
This should never happen

Reply:
Checked `addESDTEntryInVMOutput` and there should be no problem even if `esdtDataBytes` is nil